createdisk.sh

```diff
@@ -54,14 +54,24 @@ function sparsify {
             echo "${GUESTFISH} failed to start, aborting"
             exit 1
     fi
-
-    guestfish --remote <<EOF
+    
+    if [ ${OKD_INSTALL} == false ]; then
+        guestfish --remote <<EOF
 add-drive $baseDir/$srcFile
 run
 luks-open $partition coreos-root
 mount /dev/mapper/coreos-root /
 zero-free-space /boot/
 EOF
+    else
+        guestfish --remote <<EOF
+add-drive $baseDir/$srcFile
+run
+mount $partition /
```

Comment:
@anjannath if it is same partition then same might work for RCOS we can try with `mount $partition` on RCOS also.

Reply:
But RHCOS still has luks encryption, its only fcos that doesn't have it

Reply:
I believe I would add one layer of indirection here,
```
if [ ${OKD_INSTALL} == false ]; then
   use_luks=false
else
  use_luks=true
fi
```
and then check `use_luks` in the guestfish code.
You probably can split the `guestfish --remote` invocation to keep the common part
```
guestfish --remote <<EOF
add-drive $baseDir/$srcFile
run
EOF

if [ -n "${use_luks}" ]; then
    guestfish --remote luks-open $partition coreos-root
fi
guestfish --remote <<EOF
mount /dev/mapper/coreos-root /
zero-free-space /boot/
EOF
```
```